src/amo/components/Link.js

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+
+export class LinkBase extends React.Component {
+  static propTypes = {
+    base: PropTypes.string,
+    children: PropTypes.node,
+    to: PropTypes.object.isRequired,
```

Comment:
Oh, yeah, I figured `object` was the catch-all. I just looked up exactly how `PropTypes` work and will fix.

`Link` actually allows undefined `to` prop that will not output a `href`... dunno what type that should be.